Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -39,8 +41,10 @@ FLOW.QuestionAnswerView = Ember.View.extend({
     questionId = this.content.get('questionID');
     q = FLOW.store.find(FLOW.Question, questionId);
     type = q.get('type');
+    multipleFlag = q.get('allowMultipleFlag');
     this.set('isTextType', type == 'FREE_TEXT' || type === 'CASCADE');
     this.set('isOptionType', type == 'OPTION');
+    this.set('isMultipleOptionType', multipleFlag === true);
```

Comment:
Since `q.get('allowMultipleFlag')` is already a boolean, couldn't you could probably just assign it directly?